Azure/sonic-buildimage
# [201911][Mellanox][SAI] update submodule pointer

* Open ACL Outer VLAN ID for egress for ports part of VLAN RIF

Signed-off-by: Volodymyr Samotiy <volodymyrs@nvidia.com>



#### Why I did it
 Open ACL Outer VLAN ID for egress for ports part of VLAN RIF

#### How I did it
Updated SAI submodule pointer

#### How to verify it
Build an image, deploy and check all is up and running.
Verify ACL sonic-mgmt test is passing

#### Which release branch to backport (provide reason below if selected)




#### Description for the changelog

[201911][Mellanox][SAI] update submodule pointer

#### A picture of a cute animal (not mandatory but encouraged)
